classes/DirtEvent.sc

```diff
@@ -84,8 +84,9 @@ DirtEvent {
 
 		if (~unit == \c) { speed = speed * ~unitDuration * ~cps };
 
-		endSpeed = speed * (1.0 + (accelerate.abs.linexp(0.01, 4, 0.001, 20, nil) * accelerate.sign));
-		if(endSpeed.sign != speed.sign) { endSpeed = 0.0 }; // never turn back
+		// endSpeed = speed * (1.0 + (accelerate.abs.linexp(0.01, 4, 0.001, 20, nil) * accelerate.sign));
+		// if(endSpeed.sign != speed.sign) { endSpeed = 0.0 }; // never turn back
+		endSpeed = speed * (1.0 + accelerate);
```

Comment:
I would find `accelerate "1"` == no acceleration to be very bizarre.

The "classic" definition is that for every sample, playback speed is increased by accelerate/samplerate.  So `accelerate "1"` means `speed` increases at one unit per second (for the default `unit "r"`).  Maybe it's best to recreate that?